resources/pictures/docicons/osx/genicons.py

```diff
@@ -0,0 +1,111 @@
+#
+# genicons.py
+#
+# MilkyTracker document icons generation script for OSX
+# Dale Whinham 25/7/2015
+#
+# To run this script you will need:
+#
+# Python 2.7:
+#     brew install python
+#
+# PyObjC:
+#     pip install -U pyobjc-core
```

Comment:
Ooh! I totally forgot OSX comes with Python 2.7, and it looks like Apple provide PyObjC aswell. I asked some colleagues for the output of `find /System/Library/Frameworks/Python.framework | grep PyObjC`, and a Mountain Lion user was getting tons of results, so I think we're good!

I've been using the Homebrew version, so PyObjC had to be added to that installation. You'll only need to install PyObjC if your default Python is something other than the preinstalled one.

I'll amend the documentation.